README.md

```diff
@@ -10,15 +10,6 @@
   <img src="./website/static/img/readme_diagram.svg" alt="Vector">
 </p>
 
----
-
-<p align="center">
-  <strong>
-    New post! <a href="https://vector.dev/blog/how-we-test-vector/">How We Test Vector</a>
-  </strong>
-</p>
-
----
```

Comment:
Was this automatically dropped when running `make generate`? This seems an odd removal.